### DISCUSSION TITLE
Featuressouite for all user stories

### DISCUSSION THREAD

**Discussion by Sandbye** [SoftMain23/JHotDrawSM23#8] [Announcements]
**SHORT NOTE**
We are apparently, supposed to make it so that all the traces happens in one tracer folder.
That means we have to have it on one branch. 
I have taken it into action myself, such that the **/dev** branch, will be used for this.

# TODO

**Features to trace**
#1 #3 #5 #6 #7 

## Information needed

If you currently have already, found the correct place to implement and use the *FeatureEntryPoint* then write a comment which contains:

---- _Start_

1. Module (e.g. jhotdraw-core)
2. Folder path (e.g. org/jhotdraw/draw/tool/DefaultDragTracker.java) // Use source path)
3. File name
4. Method(s) to track

---- _End_

Thanks

**Comment by DunkyKanstrup** [comment]
jhotdraw-core
src/main/java/org/jhotdraw/draw/tool/TextAreaCreationTool.java
TextAreaCreationTool
beginEdit and GetFieldBounds

**Comment by sabildm** [comment]
jhotdraw-core
src/main/java/org/jhotdraw/draw/action/GroupAction.java
GroupAction
GroupAction, groupFigures

**Comment by Kund0002** [comment]
jhotdraw-gui
src/main/java/org/jhotdraw/gui/plaf/palette/PaletteToolBarUI.java
FontPaletteToolbarUI
mousepressed, mousereleased, mousedragged
